src/test/ui/rfc-2397/feature-gate-do_not_recommend.rs

```diff
@@ -0,0 +1,10 @@
+// check-pass
+
+#![feature(do_not_recommend)]
+
+#[do_not_recommend]
+trait Foo {
+}
```

Comment:
I think this should be a test that actually will change behaviour when the attribute gets an implementation in the future

Reply:
Replaced with a more complete workflow.